content/docs/faq-ajax.md

```diff
@@ -88,7 +88,7 @@ Here is the equivalent with [Hooks](https://reactjs.org/docs/hooks-intro.html):
 ```js
 function MyComponent() {
   const [error, setError] = useState(null);
-  const [isLoaded, setIsLoaded] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(true);
```

Comment:
It can't start as "is loaded" since we need to load first. The variable name should be read as a question: "is the data loaded?". Hope this clarifies why this change would worsen readability